src/engine/renderer/tr_bsp.cpp

```diff
@@ -6579,31 +6575,34 @@ void R_BuildCubeMaps()
 		VectorClear( cubeProbe->origin );
 	}
 
-	Log::Notice("...pre-rendering %d cubemaps", tr.cubeProbes.size() );
-	Log::Notice("0%%  10   20   30   40   50   60   70   80   90   100%%" );
-	Log::Notice("|----|----|----|----|----|----|----|----|----|----|" );
+	Log::Notice( "...pre-rendering %d cubemaps", tr.cubeProbes.size() );
+	Log::Notice( "0%%  10   20   30   40   50   60   70   80   90   100%%" );
+	Log::Notice( "|----|----|----|----|----|----|----|----|----|----|" );
```

Comment:
I guess you can remove the `0%…100%` lines and only keep the “pre-rendering …” one.

We had similar noisy log for other things like compiling shaders but it was maybe removed 10 years ago. I guess this one is left because the feature was dead and unused from even before.

Reply:
I guess the initial idea was to write a progression bar, but our `Log` functions always adds an `endline` character at the end of the log string, so this will not work anyway…

Reply:
True. In fact, you can't even see any of those logs until it's done rendering the cubemaps, so I've removed that now.